src/Microsoft.Sbom.Contracts/Contracts/SpdxMetadata.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Sbom.Contracts;
 /// <summary>
 /// The object representation of all the metadata in an SPDX document.
 /// </summary>
-public class SpdxMetadata
+public abstract class SpdxMetadata
```

Comment:
Will this be abstract once the breaking changes are merged? If not, should it just be non-abstract now?

Reply:
No it won't so we can just make it non-abstract

Reply:
Yeah when I originally made the changes that this PR are based on I was under the impression that we did not want to make any breaking changes at all, so I was experimenting with how to do that. But if this is just a temporary change and we plan on making breaking changes in a future release, then I agree that it makes sense to leave this as non-abstract and we can probably also delete `Spdx30Metadata`